include/deal.II/base/config.h.in

```diff
@@ -267,6 +267,23 @@
 
 
 /***********************************************************************
+ * A macro that is used to mark function as not throwing any exceptions.
+ * The appropriate way to do this in C++98 was to say
+ *     void foo () throw ();
```

Comment:
```throw()``` cannot throw see http://stackoverflow.com/questions/12833241/difference-between-c03-throw-specifier-c11-noexcept and http://stackoverflow.com/questions/88573/should-i-use-an-exception-specifier-in-c